database/frontend/illustrations.json

```
@@ -1,50 +1,86 @@
+    "url": "https://drawkit.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "ManyPixels",
+    "description": "Many pixel illustration 2500+ royalty free illustrations to power up your designs.",
+    "url": "https://www.manypixels.co/gallery/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Reshot",
+    "description": "Free Icons & Illustrations design freely with instant downloads and commercial licenses.",
+    "url": "https://www.reshot.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "StorySet",
+    "description": "Awesome free customizable illustrations for your next project.",
+    "url": "https://storyset.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Freepik",
+    "description": "Freepik goal is to offer the best high-quality content: illustrations, photos, icons, mockups, and presentations templates etc.",
+    "url": "https://www.freepik.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Blush Design",
+    "description": "Blush design is a creative platform offering a diverse collection of customizable illustrations for designers, developers, and content creators. With an extensive library of vibrant artwork, users can effortlessly enhance their projects. The platform allows easy customization of colors, styles, and elements to suit individual needs, empowering creativity and enhancing visual communication.",
+    "url": "https://blush.design/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "UI Store Design",
+    "description": "UI store design is a platform dedicated to providing a wide range of design resources for UI/UX professionals. It offers a curated collection of high-quality templates, icons, illustrations, and other assets to enhance the visual appeal and user experience of digital products. With a focus on simplicity and efficiency.",
+    "url": "https://www.uistore.design/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Skribbl",
+    "description": "Enrich your creative projects and support free access to art of education with the collection of hand-drawn illustrations.",
+    "url": "https://weareskribbl.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "Humaaans",
+    "description": "Humaaans is a flexible illustration library that simplifies visual storytelling. With diverse customizable human figures, it offers a wide range of poses and clothing options for inclusive and unique illustrations. Create captivating visuals effortlessly for websites, presentations, and more. Customize, mix, and match vector illustrations to bring your stories to life.",
```

[praise] Great job! :) This describes the benefits of using this source in a detailed way! 👍🏾 
```suggestion
    "description": "Humaaans is a flexible illustration library that simplifies visual storytelling. With diverse customizable human figures, it offers a wide range of poses and clothing options for inclusive and unique illustrations. Create captivating visuals effortlessly for websites, presentations, and more. Customize, mix, and match vector illustrations to bring your stories to life.",
```